spec/models/internal_post_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+describe InternalPost do
+  it { should belong_to :user }
+  it { should have_many :comments }
+
+  # TODO
+  # it { should have_field(:title) }
```

Comment:
We can remove these `should have_field` comments. I think it is okay to only test `validate_presence_of ...`